android-static-binding-generator/project/build.gradle

```diff
@@ -14,6 +14,15 @@ def cachedJarsFilePath = "$projectDir/cached.txt"
 def jsParserP = "$projectDir/parser/js_parser.js"
 def jsFilesParametersP = "$projectDir/jsFilesParameters.txt"
 
+def webpackWorkersExcludePath = "$projectDir/../../src/main/assets/app/__worker-chunks.json"
+def webpackWorkersExcludesList;
+
+def workersExcludeFile = file(webpackWorkersExcludePath);
+def filterWorkerFiles = false;
+if (workersExcludeFile.exists()) {
+	filterWorkerFiles = true;
+	webpackWorkersExcludesList = new JsonSlurper().parseText(workersExcludeFile.text)
+}
```

Comment:
if the `workersExcludeFile` doesn't exist, please add some kind of warning or error